perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -174,17 +184,19 @@ def _record_time_of_operation(command, path, num_samples) -> list:
     num_samples: Number of times to run the command.
 
   Returns:
-    A list containing the latencies of operations in milisecond.
+    A list containing the start and end times of operations.
   """
 
   result_list = []
+
   for _ in range(num_samples):
     start_time_sec = time.time()
     subprocess.call('{} {}'.format(command, path), shell=True,
                     stdout=subprocess.DEVNULL,
                     stderr=subprocess.STDOUT)
     end_time_sec = time.time()
-    result_list.append((end_time_sec-start_time_sec)*1000)
+    result_list.append([start_time_sec, end_time_sec])
```

Comment:
Please always try to give specific name to variable, instead of giving a generic name.

Reply:
Sure! Thanks!